configs/olcf/config.GMPAS-IAF_oRRS18to6v3.titan

```diff
@@ -53,7 +53,7 @@ baseDirectory = /dir/to/analysis/output
 # option:
```

Comment:
This one is missing a few things (mind you, I haven't had a chance to test this yet):
- `mpasMeshName`
- there is the piece with `Directory with mapping files`, but not this newer part:
```
# Directory for region mask files
regionMaskDirectory = 
```
- these lines before line 37:
```
# provide an absolute path to put HTML in an alternative location (e.g. a web
# portal)
# htmlSubdirectory = /ccs/proj/cli115/www/USERNAME/RUNNAME
htmlSubdirectory = html
```

Reply:
I do see the `regionMaskDirectory` part in my version at the bottom.  I have changed the rest as you suggested.